paper.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Qiskit Dynamics: A Python package for simulating the time dynamics of quantum systems'
+tags:
+  - Python
+  - quantum
+  - quantum computer
+  - pulse
+  - control
+authors:
+  - name: Daniel Puzzuoli
+    corresponding: true
+    affiliation: 1
+  - name: Christopher J. Wood
+    affiliation: 2
+  - name: Daniel J. Egger
+    affiliation: 3
+  - name: Benjamin Rosand
+    affiliation: 4
+  - name: Kento Ueda
+    affiliation: 5
+  - name: Ian Hincks
+    affiliation: 6
+  - name: Haggai Landa
+    affiliation: 7
+  - name: Moein Malekakhlagh
+    affiliation: 2
+  - name: Avery Parr
+    affiliation: 8
+  - name: R. K. Rupesh
+    affiliation: 9
+  - name: Matthew Treinish
+    affiliation: 2
+affiliations:
+ - name: IBM Quantum, IBM Canada, Vancouver, BC, Canada
+   index: 1
+ - name: IBM Quantum, IBM T.J. Watson Research Center, Yorktown Heights, NY, USA
+   index: 2
+ - name: IBM Quantum, IBM Research Europe - Zurich, Ruschlikon, Switzerland
+   index: 3
+ - name: Department of Physics, Yale University, New Haven, CT, USA
+   index: 4
+ - name: IBM Quantum, IBM Research Tokyo, Tokyo, Japan
+   index: 5
+ - name: IBM Quantum, IBM Canada, Markham, ON, Canada
+   index: 6
+ - name: IBM Quantum, IBM Research Israel, Haifa, Israel
+   index: 7
+ - name: Department of Physics, Harvard University, Cambridge, MA, USA
+   index: 8
+ - name: School of Physics, University of Hyderabad, Hyderabad, India
+   index: 9
+
+
+date: 31 July 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+Qiskit Dynamics is an open source Python library for numerically simulating the time dynamics of finite-dimensional quantum systems. The goal of the package is to provide flexible configuration of the numerical methods used for simulation: general tools for transforming models of quantum systems for more efficient simulation (rotating frames and the rotating wave approximation), choice of array representations (dense v.s. sparse, and different array libraries), and access to different types of underlying solvers (standard ODE v.s. geometric solvers). The package also contains advanced functionality for computing time-dependent perturbation theory expressions used in robust quantum control optimization `[@perturb1; @perturb2]`. 
+
+As part of the Qiskit Ecosystem (https://qiskit.org/ecosystem), the package interfaces with other parts of Qiskit `[@Qiskit]`. Most notably, Qiskit Dynamics provides tools for simulating control sequences specified by Qiskit Pulse `[@alexander_qiskit_2020]`, which is used to specify hardware-level control of quantum computers. Higher level interfaces allow users to build and interact with simulation-based objects that target the same constraints (coupling map, timing, etc.) as a specified IBM Quantum computer.
```

Comment:
```suggestion
As part of the Qiskit Ecosystem (https://qiskit.org/ecosystem), the package interfaces with other parts of Qiskit `[@Qiskit]`. Most notably, Qiskit Dynamics provides tools for simulating control sequences specified by Qiskit Pulse `[@alexander_qiskit_2020]` which gives a pulse representation of quantum circuit instructions. Higher level interfaces allow users to build and interact with simulation-based objects that target the same constraints (coupling map, timing, etc.) as a specified IBM Quantum computer.
```
I find the last sentence a bit hard to follow but don't have a concrete suggestion. What is meant here: `Higher level interfaces allow users to build and interact with simulation-based objects that target the same constraints (coupling map, timing, etc.) as a specified IBM Quantum computer.`

Reply:
Basically we're trying to describe the `DynamicsBackend` without having to explain what a `Backend` is.

Reply:
Maybe something like:

```
By specifying a physical model for a quantum computer, users can construct an object imitating a ``backend'' object in Qiskit, which is the interface used to submit jobs to real IBM quantum computers. Using this object, users can submit jobs (in the form of quantum circuits or Qiskit Pulse schedules) and receive simulation-based results in the same format as they would with a real IBM quantum computer.
```

Reply:
Ok I see. Here is a suggestion. I think it can be improved, perhaps it gives some inspiration?
```
This enables one to build a simulator of a quantum computer that time-evolves user provided pulses, encapsulated in quantum circuit instructions, according to a Hamiltonian describing the computer. This representation is closer to the physical dynamics of the quantum computer than a simulator based on gate-level unitary matrices.
```